tests/test_datasets.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import pandas as pd
+import torchvision.transforms as T
+from quickvision.datasets.classification import CSVDataset
+
+df = pd.read_csv('tests/assets/csv_dataset.csv')
+data_dir = 'tests/assets/'
+tfms = T.Compose([
+    T.ToTensor(),
+])
+
+
+class DatasetsTester(unittest.TestCase):
+    def test_csv_dataset(self):
+        flag = False
+        complete_dataset = CSVDataset(df, data_dir, 'Image', 'Label', tfms, 'png')
+        complete_dataset[0]
```

Comment:
you can do `self.assertEquals(torch.utils.data.Dataset, complete_dataset)`

Reply:
It says this `AssertionError: <class 'torch.utils.data.dataset.Dataset'> != <quickvision.datasets.classification.CSVDataset object` when I do that.